src/xcode/ENA/ENA/Source/Scenes/OptionGroup/DatePickerOption/DatePickerDayViewModel.swift

```diff
@@ -0,0 +1,131 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import UIKit
+import Combine
+
+class DatePickerDayViewModel {
+
+	// MARK: - Init
+
+	init(
+		datePickerDay: DatePickerDay,
+		onTapOnDate: @escaping (Date) -> Void,
+		isSelected: Bool = false,
+		calendar: Calendar = .gregorian()
+	) {
+		self.datePickerDay = datePickerDay
+		self.onTapOnDate = onTapOnDate
+		self.isSelected = isSelected
+		self.calendar = calendar
+
+		switch datePickerDay {
+		case .future(let date), .past(let date), .today(let date):
+			self.date = date
+		}
+
+		update()
+	}
+
+	// MARK: - Internal
+
+	var isSelected: Bool {
+		didSet {
+			update()
+		}
+	}
+
+	let fontSize: CGFloat = 16
```

Comment:
I'm a little confused, wouldn't something like the font-size be rather part of the view than the view model?

Reply:
Yeah I can see that, but where do you draw the line? The color, text and basically everything that styles the label comes from the ViewModel why not the fontSize as well? :)

Reply:
If I understand correctly the rest of the attributes are dynamic, while this one is static - I agree that drawing a line there is difficult, but I wanted to double check whether you have had thoughts about doing it like this :-) Which you did, as I see.